### PR TITLE
D604149: Release Build Failed - buildenv-dotnet-mono-image main - Failed

### PR DESCRIPTION
**Pull request by markmccallion28**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=604149

### CODE REVIEW

File: src/main/docker/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ FROM ${DOCKER_HUB_PUBLIC}/cafapi/prereleases:buildenv-dotnet-1.1.0-SNAPSHOT
 # Copy in zypper repo key
 COPY dev_zypper_repo_key.pub .
 
-RUN zypper -n addrepo https://download.opensuse.org/repositories/home:zkrd2:mono/15.4/home:zkrd2:mono.repo && \
+RUN zypper -n addrepo --no-gpgcheck https://download.opensuse.org/repositories/home:/lemmy04/15.4/home:lemmy04.repo && \
```
Comment (markmccallion28) [inline src/main/docker/Dockerfile:25]: I couldn't get the signing key to be auto accepted so I had to turn off gpgchecks, is this ok? 

image

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
We can't do this.

**Comment by markmccallion28** [comment]
> We can't do this.
@dermot-hardy  Do you mean we can't turn off the gpgcheck or that we can't change where the msbuild package is being downloaded from?

**Comment by dermot-hardy** [comment]
I mean we can't turn off the GPG checks.  We could change the package provider if necessary.

**Pull request closed without merging**